lib/godmin/helpers/forms.rb

```diff
@@ -4,6 +4,11 @@ module Forms
       def form_for(record, options = {}, &block)
         super(record, { builder: FormBuilders::FormBuilder, inline_errors: false }.merge(options), &block)
       end
+
+      def select_tag(name, option_tags = nil, options = {})
```

Comment:
Hm, true that you can't pass a form builder to `form_tag`.. 

The problem I have with it is that it seems inconsistent. All the bootstrap markup overriding is provided by the form builder, so if you were to do a `text_field_tag` or something it would definitely look out of place. This feels similar to me.